Pcap++/header/DpdkDeviceList.h

```diff
@@ -76,14 +76,15 @@ namespace pcpp
 		bool m_IsInitialized;
 		static bool m_IsDpdkInitialized;
 		static uint32_t m_MBufPoolSizePerDevice;
+		static uint16_t m_MBufDataSize;
 		static CoreMask m_CoreMask;
 		std::vector<DpdkDevice*> m_DpdkDeviceList;
 		std::vector<DpdkWorkerThread*> m_WorkerThreads;
 
 		DpdkDeviceList();
 
 		bool isInitialized() const { return (m_IsInitialized && m_IsDpdkInitialized); }
-		bool initDpdkDevices(uint32_t mBufPoolSizePerDevice);
+		bool initDpdkDevices(uint32_t mBufPoolSizePerDevice, uint16_t mMbufDataSize);
```

Comment:
nit: let's call this variable `mBufDataSize`?

Reply:
Of course, I overlooked that

Reply:
The issue has been fixed in the commit: [Fix spelling consistency](https://github.com/seladb/PcapPlusPlus/pull/1344/commits/1b33e25c0b6c730711190a53bb153aff375af5e5)